hardware/edid.ksy

```diff
@@ -257,6 +267,222 @@ types:
         1: ratio_4_3
         2: ratio_5_4
         3: ratio_16_9
+  dtd_timing:
+    seq:
+      - id: pixel_clock_mod
+        type: u2
+        doc: |
+          Pixel Clock / 10,000
+      - id: horiz_active_pixels_8lsbits
+        type: u1
+        doc: |
+          Horizontal Addressable Video in pixels --- contains lower 8 bits
```

Comment:
The first parts (before `---`) can be moved into `id`, the second part is usually redundant (because its meaning is encoded into `type`).

Reply:
The first part (`Horizontal Addressable Video in pixels`) is already reflected in `id` (`horiz_active_pixels`). Do you mean I have to remove this duplication from `doc`? The second part is reflected in `id` as well (`8lsbits`) and `type` shows the size but not place in the resulting bit field.

Could you clarify your comment please?

```
- id: horiz_active_pixels_lsb
  type: u1
```

Is this how it should look like?

Reply:
@KOLANICH, bump.

Reply:
1. I meant something like the following:

```yaml
id: horiz_addressable_video_lo
# no doc here, this value is not meaningful by itself, is it?
-unit: px
```

```yaml
id: horiz_addressable_video_hi
-unit: px * 256
```

```yaml

horiz_addressable_video_hi:
  doc: "in pixels"
  -unit: px
  value: ...
```

2. I am not the one having authority to accept PRs in this repo. @generalmimon and @GreyCat are. So my advices are just advices and are not mandatory for your PR being accepted.

Reply:
@KOLANICH, thank you. These are useful recommendations, I fixed naming, added `unit`s and removed `doc`s for self-descriptive `seq` items.